web/packages/shared/components/UnifiedResources/ListView/ResourceListItem.tsx

```diff
@@ -132,55 +130,57 @@ export function ResourceListItem({
             overflow: hidden;
             text-overflow: ellipsis;
             white-space: nowrap;
+            // If there is no address, let this column take up the extra space.
+            ${!addr ? 'grid-column-end: address;' : ''}
```

Comment:
This is great, but maybe we could also check for `showLabelsButton`?
And make this wider if this is not showing. The condition would be more complex, perhaps it should be extracted to a function.
<img width="562" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/79a8a16d-15d1-4362-b33c-9fe59a290b21">